content/devdocs/references/p2p_networking.md

```diff
@@ -191,7 +260,29 @@ The following annotated hexdump shows an `ANNB` message with two
 inventory entries.  (The message header has been omitted.)
 
 {{ <highlight text> }}
-TODO: ANNB message hex dump
+AnnounceBlocksMessage:
+```
+0c 00 00 00 00 00 00 00 40 e2 01 00 00 00 00 00 
```

Comment:
No need to add full message . It's too much , especially if messages are long. Annotated message would be enough. Please remove